File: ios/Classes/FlutterArchivePlugin.m
#import "FlutterArchivePlugin.h"
#import <flutter_archive/flutter_archive-Swift.h>

@implementation FlutterArchivePlugin
+ (void)registerWithRegistrar:(NSObject<FlutterPluginRegistrar>*)registrar {
  [SwiftFlutterArchivePlugin registerWithRegistrar:registrar];
}
@end

File: darwin/flutter_archive/Sources/flutter_archive/FlutterArchivePlugin.swift
#if os(macOS)
import Cocoa
import FlutterMacOS
#elseif os(iOS)
import Flutter
#endif

public class FlutterArchivePlugin: NSObject, FlutterPlugin {
  public static func register(with registrar: FlutterPluginRegistrar) {
    SwiftFlutterArchivePlugin.register(with: registrar);
  }
}
